File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Java/Program4.java

import java.io.*;



class Program4{

	
	public static void main(String args[])throws IOException{

		
		char ch ='A';

	
		for(int itr = 0 ; itr < 4 ;itr ++){

			for(int jtr = 0 ; jtr<4-itr ; jtr++ ){
				
				System.out.printf("%c",ch+(jtr*2));
			}
			System.out.printf("\n");
			ch = ++ch;	
		}	
	}	
}	

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/C/program4.c
#include<stdio.h>


void main(){
	
	char ch ='A';

	
	for(int itr = 0 ; itr < 4 ;itr ++){

		for(int jtr = 0 ; jtr<4-itr ; jtr++ ){
			
			printf("%c",ch+(jtr*2));
		}
		printf("\n");
		ch = ch + 1;	
	}	
}	
